File: Algorithms/LinearSearch/linearsearch.java
//Contributed by - Saksham Kankani <superherosaksham@gmail.com>
import java.util.*;
import java.lang.*;
import java.io.*;

class Lsearch
{
  public static void main(String[] args) throws java.lang.Exception
  {
    Scanner sc=new Scanner(System.in);
    System.out.println("Enter length of the array");
    int n=sc.nextInt();
    int positionfound=0;
    int c=0;
    int ar[]=new int[n];
    System.out.println("Fill the array");
    for(int i=0;i<n;i++)
    {
      ar[i]=sc.nextInt();
    }
    System.out.println("Enter the number which is to be searched in the array");
    int target=sc.nextInt();
    for(int j=0;j<n;j++)
    {
      if(ar[j]==target)
      {
        positionfound=j;
        c++;
        break;
      }
    }
    if(c!=0)
    System.out.println("Found at index "+positionfound);
    else
      System.out.println("Target not found");
  }
}

    

File: Algorithms/LinearSearch/linearsearch.c
#include <stdio.h>

int main()
{
   int arr[10], i;
   int num, index = -1;
   printf("Enter 10 Numbers: ");
   for (i = 0; i < 10; i++)
      scanf(" %d", &arr[i]);

   printf("\nEnter a Number to Search: ");
   scanf(" %d", &num);
   for (i = 0; i < 10; i++)
   {
      if (arr[i] == num)
      {
         index = i;
         break;
      }
   }
   if (index != -1) {
      printf("\nFound at Index No. %d\n", index);
   }
   else {
      printf("\nNumber not found in array");
   }
   return 0;
}
